### ISSUE TITLE
End-device has difficulty joining network.

### ISSUE THREAD

**Issue by illperipherals** [helium/router#466]
Observation: The Helium Console shows that the LNS receives the JoinRequest and sends a JoinAccept via one of the Gateways that heard the end-device.  Device never receives the JoinAccept.  The response in the console appears to be immediate, and while this does not necessarily mean that the packet will be scheduled immediately at the Gateway transmitter, it does highlight a problem.

Diagnosis: The JoinAccept packet is transmitted outside the JOIN_ACCEPT_DELAY1 and JOIN_ACCEPT_DELAY2 windows.  Could the Gateway be using the RECEIVE_DELAY1 timer instead of JOIN_ACCEPT_DELAY1 timer?

LoRaWAN Compliance Requirement: After transmitting JoinRequest, the end-device waits for the expiry of JOIN_ACCEPT_DELAY1 before turning on its receiver.

End-Device Hack:

Enable Continuous Rx to find Rx timing.
Reduce the JOIN_ACCEPT_DELAY1 timer to value less than timing.
Preferred Solution: Network should adhere to JOIN_ACCEPT_DELAY1 and JOIN_ACCEPT_DELAY2 values per the LoRaWAN Regional Parameters specification.

**Comment by buzzware** [comment]
I'm seeing this too.
I have 2 identical third party devices that I can't modify or access the source code or any LoRaWAN config besides eui's and period.
1 joined a week ago and has been uplinking for several days.
I tried to join the other, but so far it has failed after many attempts.
Then I tried to rejoin the first, and it hasn't succeeded either.

I literally can't use Helium if it won't join - this issue should be treated as an emergency by the team, and I think the router needs more attention and developer resources.
